source/about-mongosync.txt

```diff
@@ -157,6 +157,28 @@ If the reverse sync is successful, ``mongosync`` enters the ``RUNNING`` state.
 The synchronization continues in the reverse direction from the original 
 sync job. 
 
+.. _mongosync-considerations:
+
+Considerations for Continuous Sync
+----------------------------------
+
+Consider the following when using ``mongosync`` for continuous synchronization:
+
+- For the destination cluster to have an up to date, consistent view of the 
+  source cluster’s data, ``mongosync`` must :ref:`commit <c2c-api-commit>` . 
+  If the source cluster shuts down before ``mongosync`` can commit, such as in 
+  a disaster scenario, the destination cluster might not have a consistent 
+  snapshot of the source data, and transactions might not appear atomically on 
+  the destination cluster.
```

Comment:
it would be great if we could add a link here to the content describing all the things that mongosync modifies on the destination (from this [thread](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/377/files#r1723941662))